timepiece/contracts/tests/test_contracts.py

```diff
@@ -1,11 +1,14 @@
 import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
This should be revised a bit; whole lib imports should be first, then partial imports:

```
import datetime
import mock

from dateutil.relativedelta import relativedelta

...
```